tests/unit/plugins/autoformat.js

```diff
@@ -15,8 +15,7 @@ import CodeBlockEditing from '@ckeditor/ckeditor5-code-block/src/codeblockeditin
 import HeadingCommand from '@ckeditor/ckeditor5-heading/src/headingcommand';
 import HeadingEditing from '@ckeditor/ckeditor5-heading/src/headingediting';
 import HorizontalLineEditing from '@ckeditor/ckeditor5-horizontal-line/src/horizontallineediting';
-import ListEditing from '@ckeditor/ckeditor5-list/src/listediting';
-import TodoListEditing from '@ckeditor/ckeditor5-list/src/todolistediting';
+import { ListEditing, TodoListEditing } from '@ckeditor/ckeditor5-list/src/';
```

Comment:
I reported this as a mini-issue: #371.

Reply:
This was addressed here already -> https://github.com/ckeditor/github-writer/pull/364/files#diff-6acbcdab41426405308090b572bb4b0fdfe8c4036ac35cc6181698d7b0c3fe4aL18-L20

Reply:
I know ;p Just wanted to have it reported.